apps/site/assets/ts/schedule/components/__tests__/ScheduleLoaderTest.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import { Provider } from "react-redux";
+import "whatwg-fetch";
```

Comment:
Might it work to do this in our Jest setup file?

Reply:
Good call, forgot this was a file